Sources/BraveTalk/BraveTalkJitsiCoordinator.swift

```diff
@@ -17,7 +17,11 @@ import JitsiMeetSDK
     return true
   }
   
-  public init() {}
+  public init() {
+    if !AppConstants.buildChannel.isPublic {
+      JitsiMeetLogger.add(BraveTalkJitsiLogHandler())
```

Comment:
Is `JitsiMeetLogger` holding this class strongly I guess? We don't have to hold onto our own ref to keep it alive?

Reply:
Thats how I understand it. It uses `CocoaLumberJack` under the hood which I did not use in the past.
https://github.com/jitsi/jitsi-meet/blob/master/ios/sdk/src/JitsiMeetLogger.m#L30
I think it adds it here https://github.com/CocoaLumberjack/CocoaLumberjack/blob/16815175e00cfa2a4d5c6adcb8c3395ea38b6bac/Sources/CocoaLumberjack/DDLog.m#L618

I can retain it in the jitsi coordinator class too if you think it's safer